### PR TITLE
added info about symlink

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8347

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @RowdyChildren, could you please have a look?

Dear @e0i, for your information

Thank you

**Comment by RowdyChildren** [comment]
Looks good to me!

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.
Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/security/threat-protection/microsoft-defender-atp/linux-support-install.md [hunk 1]
```diff
@@ -97,7 +97,9 @@ Then rerun step 2.
 4. If the above steps don’t work, check if SELinux is installed and in enforcing mode. If so, try setting it to permissive (preferably) or disabled mode. It can be done by setting the parameter `SELINUX` to "permissive" or "disabled" in `/etc/selinux/config` file, followed by reboot. Check the man-page of selinux for more details.
 Now try restarting the mdatp service using step 2. Revert the configuration change immediately though for security reasons after trying it and reboot.
 
-5. Ensure that the daemon has executable permission.
+5. If ```/opt``` directory is a symbolic link, create a bind mount for ```/opt/microsoft```.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/linux-support-install.md:100]: Please use only a single pair of MarkDown backticks for code block segments within normal text. Triple MarkDown backticks is used for vertical/multiline code blocs, where the backticks are kept on blank lines before and after the code. Horizontal segments should use mostly only a single pair of backticks:
```suggestion
5. If `/opt` directory is a symbolic link, create a bind mount for `/opt/microsoft`. 
```

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Pull request merged**